src/components/_navbar.scss

```diff
@@ -0,0 +1,128 @@
+// html, body {
+//   margin: 40px 0;
+//   padding: 0;
+// }
+
+.bar1, .bar2, .bar3 {
+  width: 2rem;
+  height: .3rem;
+  background-color: $primary;
+  margin: 6px 0;
+  transition: 0.4s;
+}
+nav {
+  .collapsible input[id^=collapsible]:checked + button .bar1 {
+    -webkit-transform: rotate(-45deg) translate(-9px, 7px);
+    transform: rotate(-45deg) translate(-9px, 7px);
+  }
+  .collapsible input[id^=collapsible]:checked + button .bar2 {
+    opacity: 0;
+  }
+  .collapsible input[id^=collapsible]:checked + button .bar3 {
+    -webkit-transform: rotate(45deg) translate(-8px, -9px);
+    transform: rotate(45deg) translate(-8px, -9px);
+  }
+}
```

Comment:
I can see a pattern here. Is there a way to further nest those selectors and reduce the duplicated code?